packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1329,6 +1329,36 @@ const GETOmnichannelContactsSchema = {
 
 export const isGETOmnichannelContactsProps = ajv.compile<GETOmnichannelContactsProps>(GETOmnichannelContactsSchema);
 
+type GETOmnichannelContactsSearchProps = PaginatedRequest<{
+	searchText: string;
+}>;
+
+const GETOmnichannelContactsSearchSchema = {
+	type: 'object',
+	properties: {
+		count: {
+			type: 'number',
+			nullable: true,
```

Comment:
If you wont accept `null` as a valid value, then we can remove the `nullable` prop. Them not being required is enough so they're optional.